spec/lib/rufo/formatter_source_specs/2.5/squiggly_heredoc.rb.spec

```diff
@@ -0,0 +1,11 @@
+#~# ORIGINAL heredoc_squiggly_extra_spaces
```

Comment:
Should we bump the ruby versions in https://github.com/ruby-formatter/rufo/blob/274aeab43db6a514ff43a62fcaadffd32819632a/.travis.yml#L4-L8

So that we can move this to the 2.3 test file?

Reply:
Yes good point @gingermusketeer